### DISCUSSION TITLE
Stars Don't Add Up

### DISCUSSION THREAD

**Discussion by RoyH11** [community/community#77883] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I realized on the Stars tab, the number don't add up to what I have starred. I am currently using my laptop to view my github. What is this number supposed to mean? 

![image](https://github.com/community/community/assets/98927448/ffb39434-b7c1-4dd3-8e8d-c8af2829d0e5)

Currently I have about 16 repos in this page showing. 

![image](https://github.com/community/community/assets/98927448/6319f6f8-6c05-467e-8577-1d94958435ed)

![image](https://github.com/community/community/assets/98927448/e7d6c798-df6c-44d1-8bd5-b2b98fed1dd0)

![image](https://github.com/community/community/assets/98927448/824d53e7-3c0e-4ff5-a7cc-d695912ba69c)

**Comment by LethSphere** [comment]
On which device are you trying it? Try checking that you are logged in correctly

**Reply by RoyH11** [reply]
I am using my laptop

**Comment by dakshithadissanayaka** [comment]
The number of starred repositories displayed on the Stars tab of your GitHub profile may not always match the exact number of repositories you have starred. This can be due to a few different reasons:

Private repositories: If you have starred private repositories, they will not be counted in the total number of starred repositories displayed on your profile. This is because private repositories are not visible to other users, and GitHub does not want to inadvertently reveal information about private repositories.

Deleted repositories: If you have starred a repository that has been deleted by its owner, it will no longer be counted in the total number of starred repositories displayed on your profile. This is because the repository no longer exists, and GitHub cannot track the star count for a deleted repository.

API limitations: The GitHub API, which is used to calculate the star count for your profile, may have occasional issues. This can sometimes lead to the star count being incorrect.

Caching issues: GitHub may sometimes cache the star count for your profile, which can prevent it from updating immediately when you star or unstar a repository. This is usually only a temporary issue, and the star count should update correctly after a few minutes.

If you are concerned about the accuracy of the star count on your profile, you can try the following:

Refresh the page: This will sometimes force GitHub to recalculate the star count.

Try using a different browser: This can sometimes resolve caching issues.

Contact GitHub support: If the issue persists, you can contact GitHub support for assistance.

I hope this helps!

**Comment by gutierrez00** [comment]
Manejar puro dominio no correos

**Comment by aYtUlgU** [comment]
Nerelerde ben bldye deyim

**Comment by Agilan989** [comment]
Yup it's great and easier

**Comment by jmfritzke** [comment]
MariDeBose,0*

**Reply by TNTurner23** [reply]
> MariDeBose,0*

Really? So what was this REALLY about?
